__tests__/app.test.js

```diff
@@ -114,7 +122,60 @@ describe('GET: /api/articles', () => {
         .get('/api/articles')
         .expect(200)
         .then(({body}) => {
-            expect(body).toBeSortedBy("created_at", {descending: true}, {coerce: true})
+            body.forEach((article) => {
+                const regex = /[^0-9]/g
+                const stringCreated = article.created_at.replace(regex, "")
+                article.created_at = stringCreated
+            })
+            expect(body).toBeSortedBy('created_at', {descending: true})
+        })
+    })
+})
+describe('GET: /api/articles/:article_id/comments',() => {
+    test('GET 200: Returns an array of comment objects with the correct properties', () => {
+        return request(app)
+        .get('/api/articles/5/comments')
+        .expect(200)
+        .then(({body}) => {
+            body.forEach((comment) => {
+                expect.objectContaining({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    article_id: expect.any(Number),
```

Comment:
Can you be more specific about the article id? We know that they should all have the id of 5